### DISCUSSION TITLE
Provides supporting symbolic keys

### DISCUSSION THREAD

**Discussion by jdalrymple** [nuxt/nuxt#16708] [Questions]
I noticed in Vue itself, the plugin provides logic supports symbolic keys, however within the nuxt3 framework, its limited to strings. Is there a reason for this?

**Comment by sevillaarvin** [comment]
I think it’s because symbols are not serializable.

**Comment by jdalrymple** [comment]
Perhaps, but it is definitely weird for it to not be consistent behaviour with Vue itself

**Reply by TheAlexLichter** [reply]
Well, on client-side, symbols can be used easily. If Nuxt would allow using symbols, then the state might not be transferrable to the client anymore and issues would arise. But maybe this could be mitigated somehow 🤔 

Feel free to create a feature request ☺️
